src/components/Header/Header.jsx

```diff
@@ -4,3 +4,14 @@
 // Create a Header function returning the HTML of header block
 
 // Add a default export statement for Header component to use it in the other files
+import React from 'react';
+import './Header.css';
+import nav from '../Navigation/Navigation';
+
+const Header = () => (
+  <header className="header">
+    {nav}
```

Comment:
it's not working now

```suggestion
    <Nav />
```

Reply:
you should use your component in this way